### ISSUE TITLE
Aseprite 1.1.1 hangs on start on Mac OS X 

### ISSUE THREAD

**Issue by lordnoriyuki** [aseprite/aseprite#943]
Using the latest trial version on Mac OS X 10.11.3 (15D21)

I get the following dialog on start up:

<img width="485" alt="screen shot 2016-01-24 at 3 09 55 pm" src="https://cloud.githubusercontent.com/assets/7322548/12539644/a82132a8-c2ac-11e5-9b3f-88349c252bf4.png">

and then it just hangs until I kill it.

I also tried the old 0.9.5 version, and I get an error message dialog immediately that says "Unable to create user interface display".

For the 1.1.1 hang, the problem report contains the following:

```
Date/Time:       2016-01-24 15:10:03 -0800
OS Version:      Mac OS X 10.11.3 (Build 15D21)
Architecture:    x86_64h
Report Version:  22

Command:         Aseprite
Path:            /Applications/Aseprite.app/Contents/MacOS/aseprite
Version:         1.1.1 (1.1.1)
Parent:          launchd [1]
PID:             471

Event:           hang
Duration:        4.00s (process was unresponsive for 31 seconds before sampling)
Steps:           40 (100ms sampling interval)

Hardware model:  iMac15,1
Active cpus:     8

Fan speed:       1200 rpm

--------------------------------------------------
Timeline format: stacks are sorted chronologically
Use -i and -heavy to re-report with count sorting
--------------------------------------------------

Heaviest stack for the main thread of the target process:
  40  start + 41 (aseprite + 14937) [0x4a59]
  40  _start + 216 (aseprite + 15154) [0x4b32]
  40  main + 121 (aseprite + 2598569) [0x27b6a9]
  40  allegro_main + 1064 (aseprite + 3052552) [0x2ea408]
  40  -[NSApplication run] + 1063 (AppKit + 557359) [0x9416112f]
  40  -[NSApplication nextEventMatchingMask:untilDate:inMode:dequeue:] + 121 (AppKit + 601995) [0x9416bf8b]
  40  -[NSApplication _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1057 (AppKit + 4841648) [0x945770b0]
  40  _DPSNextEvent + 2415 (AppKit + 604772) [0x9416ca64]
  40  AEProcessAppleEvent + 55 (HIToolbox + 247898) [0x9004c85a]
  40  aeProcessAppleEvent + 299 (AE + 31624) [0x97928b88]
  40  dispatchEventAndSendReply(AEDesc const*, AEDesc*) + 44 (AE + 31861) [0x97928c75]
  40  aeDispatchAppleEvent(AEDesc const*, AEDesc*, unsigned long, unsigned char*) + 595 (AE + 221777) [0x97957251]
  40  _NSAppleEventManagerGenericHandler + 211 (Foundation + 243672) [0x9e5d97d8]
  40  -[NSAppleEventManager dispatchRawAppleEvent:withRawReply:handlerRefCon:] + 451 (Foundation + 244189) [0x9e5d99dd]
  40  __76-[NSAppleEventManager setEventHandler:andSelector:forEventClass:andEventID:]_block_invoke + 118 (Foundation + 245368) [0x9e5d9e78]
  40  -[NSObject performSelector:withObject:withObject:] + 84 (libobjc.A.dylib + 44466) [0x93284db2]
  40  -[NSApplication(NSAppleEventHandling) _handleCoreEvent:withReplyEvent:] + 291 (AppKit + 629403) [0x94172a9b]
  40  -[NSApplication(NSAppleEventHandling) _handleAEOpenEvent:] + 783 (AppKit + 631110) [0x94173146]
  40  -[NSApplication _sendFinishLaunchingNotification] + 249 (AppKit + 643630) [0x9417622e]
  40  -[NSApplication _postDidFinishNotification] + 436 (AppKit + 644468) [0x94176574]
  40  -[NSNotificationCenter postNotificationName:object:userInfo:] + 92 (Foundation + 29125) [0x9e5a51c5]
  40  _CFXNotificationPost + 626 (CoreFoundation + 65058) [0x911f6e22]
  40  -[_CFXNotificationRegistrar find:object:observer:enumerator:] + 1715 (CoreFoundation + 1260019) [0x9131a9f3]
  40  ___CFXNotificationPost_block_invoke + 54 (CoreFoundation + 1285990) [0x91320f66]
  40  _CFXRegistrationPost + 460 (CoreFoundation + 1286700) [0x9132122c]
  40  ___CFXRegistrationPost_block_invoke + 75 (CoreFoundation + 1289307) [0x91321c5b]
  40  __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 20 (CoreFoundation + 1289492) [0x91321d14]
  40  __57-[NSNotificationCenter addObserver:selector:name:object:]_block_invoke_2 + 50 (Foundation + 1622770) [0x9e72a2f2]
  40  -[AllegroAppDelegate applicationDidFinishLaunching:] + 533 (aseprite + 3050885) [0x2e9d85]
  40  usleep + 60 (libsystem_c.dylib + 546339) [0x95624623]
  29  ??? [0xffffffff]
  29  _sigtramp + 43 (libsystem_platform.dylib + 20507) [0x978be01b]
  29  osx_signal_handler + 53 (aseprite + 3042485) [0x2e7cb5]
  29  ??? [0xffffffff]
  29  _sigtramp + 43 (libsystem_platform.dylib + 20507) [0x978be01b]
  29  osx_signal_handler + 53 (aseprite + 3042485) [0x2e7cb5]
  29  _unix_unlock_mutex + 9 (aseprite + 3048121) [0x2e92b9]
 *29  hndl_alltraps + 229 (kernel + 2031941) [0xffffff80003f0145]
 *24  user_trap + 1090 (kernel + 1911874) [0xffffff80003d2c42]
 *24  exception_triage + 496 (kernel + 920544) [0xffffff80002e0be0]
 *24  mach_exception_raise + 139 (kernel + 1225483) [0xffffff800032b30b]
 *18  mach_msg_rpc_from_kernel_body + 256 (kernel + 935184) [0xffffff80002e4510]
 *18  ipc_mqueue_receive + 74 (kernel + 833338) [0xffffff80002cb73a]
 *12  thread_block_reason + 175 (kernel + 1005615) [0xffffff80002f582f]
 *12  ??? (kernel + 1019164) [0xffffff80002f8d1c]
 *10  machine_switch_context + 316 (kernel + 1891868) [0xffffff80003cde1c]


Process:         aseprite (Aseprite) [471]
Path:            /Applications/Aseprite.app/Contents/MacOS/aseprite
Architecture:    i386
Parent:          launchd [1]
UID:             501
Task size:       4253 pages
CPU Time:        1.621s
Note:            Unresponsive for 31 seconds before sampling
Note:            2 idle work queue threads omitted

....
```

**Comment by dacap** [comment]
This is the third time that I'm contacted about this issue and the Skia port (#139) fixes it. I'll be testing the Skia port for OS X. It might be ready for v1.1.2 instead of v1.2.

**Comment by lordnoriyuki** [comment]
Is there any known workaround? I presume Aseprite works on Mac for most people and if so, any idea what might be different about their environment compared to mine?

**Comment by dacap** [comment]
Contact me at support@aseprite.org

**Comment by dacap** [comment]
It's fixed with the new backend #139
